pkgs/applications/display-managers/greetd/tuigreet.nix

```diff
@@ -16,6 +18,14 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-RkJjAmZ++4nc/lLh8g0LxGq2DjZGxQEjFOl8Yzx116A=";
 
+  nativeBuildInputs = [ scdoc ];
+
+  postInstall = ''
+    mkdir -p $out/share/man/man1
+
+    scdoc < contrib/man/tuigreet-1.scd > $out/share/man/man1/tuigreet.1
```

Comment:
Better to use `installManpage`, and add `installShellFiles` to `nativeBuildInputs`.

Reply:
Are you saying to use `installManPage` instead of file redirection from `scdoc`?

Reply:
scdoc is still needed, but the output can be installed by `installManpage` to indicate that we are installing manpage here

Reply:
Changed, please have a look and let me know if this is satisfactory.